packages/fluentui/react-northstar/src/components/Tree/Tree.tsx

```diff
@@ -26,12 +30,24 @@ import {
   ShorthandCollection,
   ComponentEventHandler,
   ObjectShorthandCollection,
+  FluentComponentStaticProps,
+  ProviderContextPrepared,
 } from '../../types';
-import { hasSubtree, removeItemAtIndex, getSiblings, TreeContext, TreeRenderContextValue } from './utils';
+import {
+  getAllSelectableChildrenId,
+  isAllGroupChecked,
+  hasSubtree,
+  removeItemAtIndex,
+  getSiblings,
+  TreeContext,
+  TreeRenderContextValue,
+} from './utils';
+// @ts-ignore
```

Comment:
To avoid this `//@ts-ignore` we could create a `global.d.ts` file with:

```typescript
declare module 'react-fela' {
  export const ThemeContext;
}

export {};
```

But I guess there's a reason why didn't happened... So why @layershifter?